tools/make/gn.toml

```diff
@@ -64,6 +60,17 @@ exit_on_error true
 cd ffi/gn
 
 rm -rf vendor
+rm -f .cargo/config
+rm -f Cargo.lock
+
+exec --fail-on-error cargo generate-lockfile
+
+# These versions should match what is in icu_capi_gn Cargo.toml
+exec --fail-on-error cargo update proc-macro2 --precise 1.0.75
```

Comment:
thought: can we specify this with an `=` bound in `Cargo.toml`?

Reply:
i guess that would end up with them getting public gn targets. hm.

Reply:
Would it work for the cargo resolver if I listed the versions as build-dependencies or dev-dependencies?